exchange/src/wrapper/messaging/comms.cpp

```diff
@@ -101,15 +103,20 @@ ExchangeProxy::publish_message(const std::string& message)
     lock.unlock();
 }
 
-std::variant<start_time, tick_update>
-ExchangeProxy::consume_message()
+algorithm_t
+ExchangeProxy::consume_algorithm() {
+    return consume_message<algorithm_t>();
+}
```

Comment:
Nit but I feel like we probably don't need this function.

Reply:
My reasoning was that `consume_message()` was private for a reason, so we should keep it that way?

Reply:
I think this is a code smell. I'm okay with it for now